app/Http/Controllers/PullRequestController.php

```diff
@@ -46,4 +46,20 @@ public function store(StorePullRequestRequest $request): RedirectResponse
 
         return to_route('pull-requests.index');
     }
+
+    /**
+     * Remove the specified resource.
+     *
+     * @param PullRequest $pullRequest
+     *
+     * @return RedirectResponse
+     */
+    public function destroy(PullRequest $pullRequest): RedirectResponse
+    {
+        abort_if($pullRequest->user_id !== auth()->id(), 403);
```

Comment:
same here, please check with policy and form request authorize.

Reply:
use a policy and replace to $this->authorize('action', model::class);